### PR TITLE
[NUI] Fix CA2000 Warnings for View

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###
Fix CA2000 Warnings for View

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by elishateng — APPROVED** [review]
LGTM

**Review by zg2nets — APPROVED** [review]
lgtm

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by huiyueun — APPROVED** [review]
Looks good to me

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 1]
```diff
@@ -2265,12 +2297,16 @@ public bool BackgroundImageSynchronosLoading
                 backgroundImageSynchronosLoading = value;
 
                 string bgUrl = null;
-                Background.Find(ImageVisualProperty.URL)?.Get(out bgUrl);
+                var pValue = Background.Find(ImageVisualProperty.URL);
+                pValue?.Get(out bgUrl);
+                pValue?.Dispose();
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:2302]: It is ok, but I think it is better to check nullity once for better readability.
```
if (pValue != null)
{
    pValue.Get(out bgUrl);
    pValue.Dispose();
}
```

**Review by rabbitfor — COMMENTED** [review]
Overall code looks good.
I think It'd be better to think of `using` statement to make code simple.

**Review by rabbitfor — APPROVED** [review]
LGTM

File: src/Tizen.NUI/src/public/BaseComponents/View.cs (hunk 1 above)
Comment (zhouleonlei) [reply src/Tizen.NUI/src/public/BaseComponents/View.cs:2298]: > It is ok, but I think it is better to check nullity once for better readability.
> 
> ```
> if (pValue != null)
> {
>     pValue.Get(out bgUrl);
>     pValue.Dispose();
> }
> ```

OK. I will do it next time

**Pull request merged**